Define _option value_ and _digit size option_ (#1039)
unicode-org/message-format-wg: spec/functions/number.md

#### Digit Size Options

Some _options_ of number _functions_ are defined to take a _digit size option_.
The _function handlers_ for number _functions_ use these _options_ to control aspects of numeric display
such as the number of fraction, integer, or significant digits.

> A **_<dfn>digit size option</dfn>_** has an _option value_ that the _function_ interprets
> as a small integer value greater than or equal to zero.

Reviewer: This change makes it not a proper term definition. "is" was better here.

```suggestion
A **_<dfn>digit size option</dfn>_** is an _option_ 
whose _option value_ is interpreted by the _function_
as a small integer greater than or equal to zero.
```
